### PR TITLE
WAZO-2099 - Migrate agid to Python 3

### PR DESCRIPTION
**Pull request by sopelj**

### CODE REVIEW

**Comment by sopelj** [comment]
Depends-on: [wazo-platform/xivo-lib-python#106](https://github.com/wazo-platform/xivo-lib-python/pull/106)

**Comment by sopelj** [comment]
recheck

**Comment by sopelj** [comment]
recheck

File: wazo_agid/modules/tests/test_call_recording.py [hunk 1]
```diff
@@ -208,28 +245,11 @@ def test_record_caller_records_when_not_an_outcall_and_all_enabled(
...
-            'WAZO_USERUUID': 'the-users-uuid',
-            'XIVO_OUTCALLID': '1',
-        }
-        self.agi.get_variable.side_effect = agi_variables.get
-        user.return_value = Mock(
-            call_record_outgoing_internal_enabled=True,
-            call_record_outgoing_external_enabled=True,
-        )
-
-        record_caller(self.agi, self.cursor, None)
```
Comment (fblackburn1) [inline wazo_agid/modules/tests/test_call_recording.py:230]: Why removing test on black commit?

Comment (sopelj) [reply wazo_agid/modules/tests/test_call_recording.py:230]: It's `black` and `flake8`. When adding them `flake8` correctly identified this test to be a duplicate, so I removed it. Not sure why it was there twice. Maybe a remnant of a merge conflict or just an accident.

Comment (fblackburn1) [reply wazo_agid/modules/tests/test_call_recording.py:230]: Oh weird ..
Thank you :+1:

**Review by fblackburn1 — APPROVED** [review]
Nice refactor
Thank you for all this cleanup :pray:
